src/shadow-embed.js

```diff
@@ -61,7 +66,15 @@ export function createShadowRoot(hostElement) {
   // Native support.
   const shadowDomSupported = getShadowDomSupportedVersion();
   if (shadowDomSupported == ShadowDomVersion.V1) {
-    return hostElement.attachShadow({mode: 'open'});
+    const shadowRoot = hostElement.attachShadow({mode: 'open'});
+    if (!shadowRoot.styleSheets) {
+      Object.defineProperty(shadowRoot, 'styleSheets', {
+        get: function() {
+          return new StyleSheetListImpl(shadowRoot);
```

Comment:
Unfortunately, everywhere we use this property is via array access. E.g. `x.styleSheets[0]`. I'm thinking maybe let's use array instead? Or any other idea?

Reply:
Ah I missed that, I was just reading the spec which only offers an items property.